doc/nrf/ug_matter_overview_dfu.rst

```diff
@@ -0,0 +1,189 @@
+.. _ug_matter_overview_dfu:
+
+Matter OTA
+##########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter Over-The-Air is a process that allows a Matter device in a Matter fabric to update its firmware.
+Most of the Matter devices will require such an update at some point during their lifecycle.
+The software image is provided by another Matter node, which acquired information about it and downloaded it from a centralized and reliable source.
+The download takes place over a special transfer protocol and user consent is required for the image to be applied on the target Matter device.
+
+.. _ug_matter_overview_dfu_roles:
+
+Matter OTA roles
+****************
+
+The following roles participate in the Matter OTA process:
+
+OTA Requestor
+   Any Matter device that is going to have its firmware updated.
+   This role corresponds to the OTA Requestor :ref:`device type <ug_matter_overview_data_model>`, which implements the client role for the OTA Software Update Provider cluster and the server role for the OTA Software Update Requestor cluster.
+
+   The OTA Requestor can only receive information about available software updates from the OTA Provider.
+   Because of this, it must implement the mandatory mechanism to discover providers and query the OTA Provider periodically about new software images using the mandatory polling mechanism.
+   However, it can also receive information about the OTA Providers from the commissioner during :ref:`ug_matter_overview_commissioning`.
+
+OTA Provider
+   The device that provides the image for the OTA update.
+   This role corresponds to the OTA Provider device type, which implements the server role for the OTA Software Update Provider cluster and the client role for the OTA Software Update Requestor cluster.
+   Any device on the fabric can assume this role, as long as it has access to the Internet or to the database with links to software images.
+
+   The OTA Provider learns about new software images using any custom mechanism provided by the device manufacturer.
+   It can also learn about new software updates from the :ref:`ug_matter_device_dcl`, if it supports querying the DCL.
+
+   After the OTA Provider learns about a new software image, it downloads it to its internal cache.
+   Then, once requested, it sends to the OTA Requestor using Matter's :ref:`ug_matter_overview_dfu_image_bdx`.
+
+   The OTA Provider can announce its presence to OTA Requestors on the fabric.
+   The OTA Requestor can also learn about the OTA Providers on the fabric during commissioning or through dynamic discovery.
+
+.. figure:: /images/matter_ota_roles.svg
+   :alt: Matter OTA roles
+
+   Matter OTA roles
+
+The following parties can also be seen as participating in the process:
+
+Image Server
+   Provides the OTA image to the OTA Provider.
+   This is usually a server set up by the vendor who creates the software image, but the image can also be provided from a centralized source, such as :ref:`ug_matter_device_dcl`.
```

Comment:
This is misleading. Maybe this:

`This is usually a server set up by the vendor who creates the software image. Information about the software image URI is then typically provided in a :ref:`ug_matter_device_dcl`.